config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml

```
@@ -764,7 +764,7 @@ presubmits:
         - --repo-root=.
         - --gcp-zone=us-west1-b
         - --parallelism=1
-        - --focus-regex=\[Serial\]
+        - '--label-filter=Feature: containsAny Serial'
```

uh @pohly are we calling [Serial] a feature now?? That's super confusing?

---

I think this should be:
```suggestion
        - '--label-filter=Serial'
```

---

For cluster-e2e, I test label filters like this:
```
_output/bin/e2e.test --ginkgo.dry-run --ginkgo.v --ginkgo.silence-skips  --ginkgo.label-filter='Deprecated'
```

SIG Testing is iterating on the labeling and we plan to communicate more about label filtering soon ...
https://groups.google.com/g/kubernetes-sig-testing/c/-qsyL_H3f7M

---

thanks for the feedback @BenTheElder . Now that the relevant lane is unbroken, I'll post a followup PR to further simplify and will integrate your suggestions.